src/kendo/Bar.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["jquery","kendo.all.min","../common/HTMLWidget"], factory);
+    } else {
+        root.kendo_Bar = factory();
+    }
+}(this, function (jquery, kendo, HTMLWidget) {
+
+
+
+    function Bar() {
+        HTMLWidget.call(this);
+
+        var context = this;
+        this._tag = "div";
+        this._type = "unknown";
+        this._chartType = "BarChart";
```

Comment:
```this._type, this._chartType```  Aren't really needed at this point (but you may end up wanting something similar in a Kendo base class when you add more charts)...